test/common/config/grpc_mux_impl_test.cc

```diff
@@ -198,8 +198,6 @@ TEST_F(GrpcMuxImplTest, TypeUrlMismatch) {
 // Validate behavior when watches has an unknown resource name.
 TEST_F(GrpcMuxImplTest, WildcardWatch) {
   setup();
-
-  InSequence s;
```

Comment:
Why remove this?

Reply:
Sorry. This was not intentional. Some merge issue master looks like